docs/source/en/perf_infer_gpu_one.md

```diff
@@ -32,6 +32,7 @@ Make sure to follow the installation guide on the repository mentioned above to
 We natively support Flash Attention 2 for the following models:
 
 - Llama
+- Mistral
```

Comment:
As I mentioned before, I think it is important to add the expected improvement from this addition. Let's add `MistralFlashAttention` to the `mistral.md` with a small section on expected gains and observed gains etc with the result and benchmark you ran. Specifying that it depends on the hardware etc etc